ffs/api/istore.go

```diff
@@ -121,6 +121,6 @@ func (s *instanceStore) getCids() ([]cid.Cid, error) {
 	return cids, nil
 }
 
-func makeCidConfigKey(c cid.Cid) datastore.Key {
-	return dsCidConfig.ChildString(c.String())
+func makeStorageConfigKey(c cid.Cid) datastore.Key {
+	return dsStorageConfig.ChildString(c.String())
```

Comment:
So, if we want, we can migrate data that is stored in here as old `CidConfig` to `StorageConfig`.

Another option would be to keep `CidConfig` as a file-private type and continue to use it as the serialization format and convert to/from it on the way in/out.

Another option is no migration at all and move forward with the new data format.

Reply:
Mentioned some of this in the slack convo.